policyengine_core/parameters/vectorial_parameter_node_at_instant.py

```diff
@@ -23,7 +23,7 @@ def build_from_node(
         node: "ParameterNode",
     ) -> "VectorialParameterNodeAtInstant":
         VectorialParameterNodeAtInstant.check_node_vectorisable(node)
-        subnodes_name = node._children.keys()
+        subnodes_name = sorted(node._children.keys())
```

Comment:
Sorry just remind me why this is necessary? Thanks

Reply:
I got this change from this commit: https://github.com/openfisca/openfisca-core/commit/c0782fa5e98bb5aab695863eed38c2856366fcc8
Without it I get the TypeError on unit test runs.